packages/material-ui/src/styles/createStyles.test.js

```diff
@@ -0,0 +1,9 @@
+import { assert } from 'chai';
+import { createStyles } from '.';
+
+describe('createStyles', () => {
+  it('is the identity function', () => {
+    const styles = {};
+    assert.strictEqual(createStyles(styles), styles);
```

Comment:
This is a little nit-picking, but does `strictEqual` compare properties or checks that the passed in arguments are the same "pointer". Meaning, the result of `createStyles` and `styles` is the same *reference*.

If not, we could add another tests that just check for `===`.

Reply:
`strictEqual` uses `===`.